File: src/main/java/org/hamster/dto/Versioning.java
package org.hamster.dto;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlType;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "Versioning", propOrder = {

})
public class Versioning {

    @XmlElement(required = true)
    protected String latest;
    @XmlElement(required = true)
    protected String release;
    @XmlElement(required = true)
    protected Versions versions;
    @XmlElement
    protected String lastUpdated;

    public Versions getVersions() {
        return versions;
    }

    public void setVersions(Versions versions) {
        this.versions = versions;
    }

    public String getLatest() {
        return latest;
    }

    public void setLatest(String latest) {
        this.latest = latest;
    }

    public String getRelease() {
        return release;
    }

    public void setRelease(String release) {
        this.release = release;
    }

    public String getLastUpdated() {
        return lastUpdated;
    }

    public void setLastUpdated(String lastUpdated) {
        this.lastUpdated = lastUpdated;
    }
}

File: src/main/kotlin/org/hamster/dto/Versioning.kt
package org.hamster.dto

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlType

@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "Versioning", propOrder = [])
data class Versioning(
    @XmlElement(required = true)
    val latest: String? = null,

    @XmlElement(required = true)
    val release: String? = null,

    @XmlElement(required = true)
    val versions: Versions? = null,

    @XmlElement
    val lastUpdated: String? = null
)